.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    labels:
+      - dependabot
+      - actions
+    schedule:
+      interval: daily
```

Comment:
nitpick (non-blocking): daily checks are probably more frequent than we need for this repo. We have [a mix](https://github.com/search?q=org%3Aactions+path%3Adependabot.yml&type=code) of daily and weekly in the actions org.

Reply:
That's a good point! I will make a note and update this across the board for all the template repos :)